qcfractal/interface/collections/collection.py

```diff
@@ -63,18 +80,16 @@ def __str__(self):
         Collection(id='5b7f1fd57b87872d2c5d0a6d', name=`S22`, client=`localhost:8888`)
         """
 
-        uid = "local"
-        if "id" in self.data:
-            uid = self.data["id"]
+        uid = self.data.id
```

Comment:
We can now just add this to line 91 directly.

Reply:
Added in upcoming commit